src/ghdcbot/bot.py

```
@@ -1743,13 +1797,19 @@ def _run_sync_and_close() -> None:
 
             await asyncio.to_thread(_run_sync_and_close)
 
-            await interaction.followup.send("✅ Sync complete! Notifications sent for new GitHub events.", ephemeral=True)
+            await status_msg.edit(
+                content="✅ Sync complete! Notifications sent for new GitHub events.",
+            )
         except Exception as exc:
             logger.exception("Sync failed", exc_info=True)
```

Ruff (0.15.7)

[warning] Logging statement has redundant `exc_info`

(G202)